src/mlpack/core/cv/metrics/f1.hpp

```diff
@@ -29,19 +31,17 @@ namespace cv {
  * of macroaveraged precision and macroaveraged recall), and then the F1 values
  * are averaged.
  *
- * In the case of binary classification (AS = Binary) positives are assumed to
- * have labels equal to 1, whereas negatives are assumed to have labels equal to
- * 0.
- *
  * In the case of multiclass classification it is assumed that there are
  * instances of every label from 0 to max(labels) among input data points.
  *
  * The returned value for F1 will be zero if both precision and recall turn out
  * to be zeros.
  *
  * @tparam AS An average strategy.
+ * @tparam PositiveClass In the case of binary classification (AS = Binary)
+ *     positives are assumed to have labels equal to this value.
  */
-template<AverageStrategy AS>
+template<AverageStrategy AS, size_t PositiveClass = 1>
```

Comment:
I'd suggest a default positive label of `0` since that's the smallest possible label.